core-web/libs/portlets/edit-ema/portlet/src/lib/edit-ema-editor/components/dot-uve-toolbar/components/dot-ema-bookmarks/dot-ema-bookmarks.component.html

```diff
@@ -1,12 +1,16 @@
 @if (store.$previewMode()) {
     <p-button
+        pTooltip="Bookmark"
+        tooltipPosition="bottom"
         (click)="toggleBookmark()"
         [icon]="bookmarked() ? 'pi pi-star-fill' : 'pi pi-star'"
         [loading]="loading()"
         styleClass="p-button-text p-button-sm p-button-rounded"
         data-testId="bookmark-button" />
 } @else {
     <p-button
+        pTooltip="Bookmark"
```

Comment:
`Bookmark | dm`

Reply:
I have a PR covering this https://github.com/dotCMS/core/pull/31067/